fluture.js

```diff
@@ -1565,6 +1606,22 @@
     return `${this._left.toString()}.finally(${this._right.toString()})`;
   };
 
+  //----------
+
+  function FutureNever(){}
+
+  FutureNever.prototype = Object.create(Future.prototype);
+
+  FutureNever.prototype._f = function FutureNever$fork(){
+    return noop;
+  };
+
+  FutureNever.prototype.toString = function FutureNever$toString(){
+    return `Future.never`;
+  };
+
+  Future.never = new FutureNever;
```

Comment:
btw is there some reason of having separate class `FutureNever` when it could be implemented like this (as there is only one instance):

```js
Future.never = object.assign(Object.create(Future.prototype),{
  _f: function FutureNever$fork(){
    return noop;
  },
  toString: function FutureNever$toString(){
    return `Future.never`;
  },
})
```